Application/Setting/Univention/Service.php

```diff
@@ -0,0 +1,623 @@
+                        $Class = $StudentSchool.'-'.$Class;
+                    }
+                    sort($ClassList);
+                    $Item['school_classes'] = implode(',', $ClassList);
+                }
+            }
+
+            if(!empty($schools)){
+                $schools = array_unique($schools);
+            }
+
+            $UploadItem['schools'] = $schools;
+
+            array_push($activeAccountList, $UploadItem);
+        });
+
+        return $activeAccountList;
+    }
+
+    /**
+     * @param bool $StudentWithoutAccount
+     *
+     * @return false|array
+     */
+    public function getExportAccount($StudentWithoutAccount = true)
+    {
+
+        $Acronym = Account::useService()->getMandantAcronym();
+        $tblAccountList = Univention::useService()->getAccountAllForAPITransfer();
+
+        $UploadToAPI = array();
+        $TeacherClasses = array();
+        $TeacherSchools = array();
+
+        $tblYear = Term::useService()->getYearByNow();
+        if ($tblYear){
+            $tblYear = current($tblYear);
+            // Lehraufträge
+            if(($tblDivisionList = Division::useService()->getDivisionByYear($tblYear))){
+                foreach($tblDivisionList as $tblDivision){
+                    if(($tblDivisionSubjectList = Division::useService()->getDivisionSubjectByDivision($tblDivision))){
+                        foreach($tblDivisionSubjectList as $tblDivisionSubject){
+                            if(($tblDivisionTeacherList = Division::useService()->getSubjectTeacherByDivisionSubject($tblDivisionSubject))){
+                                foreach($tblDivisionTeacherList as $tblDivisionTeacher){
+                                    if(($tblPersonTeacher = $tblDivisionTeacher->getServiceTblPerson())){
+                                        $SchoolString = '';
+                                        // wichtig für Schulgetrennte Klassen (nicht Mandantenweise)
+                                        if(($tblCompany = $tblDivision->getServiceTblCompany())
+                                            && Consumer::useService()->isSchoolSeparated()){
+                                            if(($tblSchoolType = $tblDivision->getType())){
+                                                $tblSchoolTypeString = Type::useService()->getSchoolTypeString($tblSchoolType);
+                                                $SchoolString = $Acronym.$tblSchoolTypeString.$tblCompany->getId();
+                                                $TeacherSchools[$tblPersonTeacher->getId()][$tblCompany->getId().'_'.$tblSchoolTypeString] = $SchoolString;
+                                                $SchoolString .= '-';
+                                            }
+                                        }
+                                        $TeacherClasses[$tblPersonTeacher->getId()][$tblDivision->getId()] = $SchoolString.$tblDivision->getTblLevel()->getName().$tblDivision->getName();
```

Comment:
Codacy found an issue: [Line exceeds maximum limit of 180 characters; contains 186 characters](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)